src/runtime/dom-particle.js

```diff
@@ -112,8 +112,8 @@ export class DomParticle extends XenStateMixin(DomParticleBase) {
       }
     }
     // make sure we invalidate once, even if there are no incoming handles
-    setTimeout(() => !this._hasProps && this._invalidate(), 200);
-    //this._invalidate();
+    //setTimeout(() => !this._hasProps && this._invalidate(), 200);
```

Comment:
Double checking -- do we need this change, if so why (add/update comment?).

Reply:
Also should remove commented out code if we don't need it.

Reply:
A particle needs to `_invalidate` at least once. Normally, this is done when props change (`_handlesToProps`), but It's not (directly) a priori known to dom-particle whether or not `_handlesToProps` will ever be called. It's not obvious what the performance trade-off is here (if we go async, we may avoid unnecessary validation but also might just be wasting cycles). 
I chose to avoid the asynchrony because it made pipes-demo faster, but it strikes me it's the wrong tradeoff in general (pipes-demo particles are unusually trivial).
I'll make a note in the code.